ern-container-gen-ios/src/hull/ElectrodeContainer.xcodeproj/project.pbxproj

```diff
@@ -392,6 +392,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 22CB683D20C9C46A0031A349 /* Project-QADeployment.xcconfig */;
 			buildSettings = {
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```

Comment:
I'm not that familiar with the `project.pbxproj` files, so just checking: Why is this set here and not in the `*.xcconfig` files that we already have (src/hull/Config)? Do we still need them in the other files? It would be good to declare it in _one_ location to avoid confusion